ocs_ci/ocs/resources/pod.py

```diff
@@ -3955,3 +3956,66 @@ def get_prometheus_pods(
     pods_with_label_match = get_pods_having_label(prometheus_label, namespace)
     prometheus_pod_objs = [Pod(**prometheus) for prometheus in pods_with_label_match]
     return prometheus_pod_objs
+
+
+def get_ceph_tool_pods(ceph_tool_label=constants.TOOL_APP_LABEL, namespace=None):
```

Comment:
similar function already exists - https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pod.py#L760

Reply:
Yes, I will change the name prefix. The problem with the function you mentioned is that it performs extra conditions and does not just get all the ceph tool pods.

Reply:
why not fixing/adjusting the existing function? Having 2 functions for similar purpose isn't ideal

Reply:
We can do that. The problem is that we must be more careful as this function is widely used in our tests.

Reply:
yes, changing it would require extensive validation. This doesn't mean we should have a dup function which does similar operation

Reply:
Okay, I will make the changes that are needed.